### PR TITLE
Increase Bio.Nexus.Tree merge_with_support testing

### PR DESCRIPTION
**Pull request by Gasta88**
This pull request addresses request made in PR #1390

### CODE REVIEW

File: Tests/test_Nexus.py [hunk 1]
```diff
@@ -494,6 +494,42 @@ def test_TreeTest2(self):
         tree.collapse_genera()
         self.assertEqual(tree.all_ids(), [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 13, 14, 17])
 
+    def test_TreeTest3(self):
```
Comment (peterjc) [inline Tests/test_Nexus.py]: The method name could be something like ``test_merge_with_support``?

File: Tests/test_Nexus.py [hunk 2]
```diff
@@ -494,6 +494,42 @@ def test_TreeTest2(self):
...
+        """Test merge_withSupport and consensus method."""
```
Comment (peterjc) [inline Tests/test_Nexus.py]: Typo, ``merge_withSupport`` --> ``merge_with_support``.

Does this indirectly test the consensus method?

Comment (Gasta88) [reply Tests/test_Nexus.py]: Yes it does.

**Comment by peterjc** [comment]
You've accidentally included ``Tests/test_EBI_ebisearch.py`` on this branch - can you do ``git rm Tests/test_EBI_ebisearch.py`` and update the branch please?

(Don't work about having a slightly messy history of this branch, I intend to squash this into one commit when I merge it)

**Comment by Gasta88** [comment]
I was looking for that file for 15 minutes in its branch, I didn't realized I placed it by mistake on another.

My fault, sorry. I'll be more careful in the future :)

**Comment by peterjc** [comment]
Thanks - we all make mistakes which is why automated testing and having someone else review your changes is a good idea.

**Pull request merged**